recipes/star-fusion/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
       - perl-io-gzip
       - perl-json-xs
       - perl-uri
-      - star <=2.5
+      - star >=2.6
```

Comment:
the issue here is that star version is 2.6.0b, and in conda eyes, 2.6.0b < 2.6.

so to fix this we could use instead, `star >2.5`